code/game/objects/items/implants/implantuplink.dm

```diff
@@ -21,3 +21,10 @@
 
 /obj/item/implant/uplink/precharged
 	starting_tc = 10
+
+/obj/item/implanter/uplink/precharged/mining
+	name = "implanter (precharged mining uplink)"

```

Comment:
why would it specifically say 'mining' 

Make it something more sensible, like 

```suggestion
	name = "Emergency uplink implanter"
```